science/ncbi_tools/Portfile

```diff
@@ -1,51 +1,105 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
 PortSystem 1.0
 
 name            ncbi_tools
-version         20080302
-description     blast is a set of tools for doing nucleotide and protein searches
-maintainers     nomaintainer
 categories      science
 platforms       darwin
+license         public-domain
+maintainers     nomaintainer
+
+version         20120620
+revision        0
+
+description     Blast is a set of tools for doing nucleotide and protein searches
+long_description \
+                "${description}"
```

Comment:
Note that you don't need quotation marks here.

Reply:
That was me, old habits die hard...